<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloneFromGitHubDialog</class>
 <widget class="QDialog" name="CloneFromGitHubDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clone from GitHub</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="HyperLinkLabel" name="label_hyperlink">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_ssh">
       <property name="text">
        <string>ssh</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ReadOnlyLineEdit" name="lineEdit_ssh"/>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_http">
       <property name="text">
        <string>http</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ReadOnlyLineEdit" name="lineEdit_http"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReadOnlyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ReadOnlyLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>HyperLinkLabel</class>
   <extends>QLabel</extends>
   <header>HyperLinkLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>CloneFromGitHubDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CloneFromGitHubDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
